IonDotnet/IIonWriter.cs

```diff
@@ -96,5 +96,106 @@ public interface IIonWriter : IValueWriter, IDisposable
         /// </summary>
         /// <param name="annotations">Set of annotations.</param>
         void SetTypeAnnotations(IEnumerable<string> annotations);
+
```

Comment:
Not a big issue, but there is another write method ``void WriteValues(IIonReader reader);``
Maybe we can reorg the methods to have all the Write... methods together

Reply:
Done. I moved the `SetTypeAnnotations` to the bottom of the file, with all the other methods relating to annotations.